griptape/rules/rule.py

```diff
@@ -5,9 +5,9 @@
 from griptape.rules import BaseRule
 
 
-@define(frozen=True)
+@define()
```

Comment:
Maybe remove parenthesis for consistency, obviously this is a really nit picky comment.